src/packages/default/FileManager/scheme.html

```diff
@@ -57,10 +57,25 @@
             </gui-menu-entry>
           </gui-menu>
         </gui-menu-bar-entry>
-
-      </gui-menu-bar>
-    </gui-vbox-container>
-
+  </gui-menu-bar>
+  <!-- CONTEXT MENU -->
+  </gui-vbox-container>
```

Comment:
I'm adding automatic scheme validation soon so users get notified automatically about this.